components/script/dom/htmltextareaelement.rs

```diff
@@ -263,6 +263,11 @@ impl HTMLTextAreaElementMethods for HTMLTextAreaElement {
         self.upcast::<Node>().dirty(NodeDamage::OtherNodeDamage);
     }
 
+    // https://html.spec.whatwg.org/multipage/#dom-textarea-textlength
+    fn TextLength(&self) -> u32 {
+       self.textinput.borrow().get_content().len() as u32
```

Comment:
Since this API says "JavaScript string length", which means [this](https://html.spec.whatwg.org/multipage/form-elements.html#dom-textarea-textlength), we should return [`self.textinput.utf16_len()`](https://doc.servo.org/script/textinput/struct.TextInput.html#method.utf16_len) instead. We should also check if this difference is covered by any tests.

Reply:
I am going to see if this try will get some tests.

Reply:
ah.. Since the code is wrong. I guest failures are still failures.

Reply:
Hopefully if there are some new tests passing they will also contain tests that will fail if the utf16 length is different than the utf8 length. We can wait and see the results before investigating test coverage :)